cmd/juju/caas/export_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package caas
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/juju/api"
+	"github.com/juju/juju/api/base"
+	"github.com/juju/juju/caas/kubernetes/clientconfig"
+	"github.com/juju/juju/cmd/modelcmd"
+	"github.com/juju/juju/jujuclient"
+)
+
+func NewAddCAASCommandForTest(cloudMetadataStore CloudMetadataStore, fileCredentialStore jujuclient.CredentialStore, clientStore jujuclient.ClientStore, apiRoot api.Connection, newCloudAPIFunc func(base.APICallCloser) CloudAPI, newClientConfigReaderFunc func(string) (clientconfig.ClientConfigFunc, error)) cmd.Command {
```

Comment:
For functions with many args like this, we typically format like so: \
\
func NewAddCAASCommandForTest( \
&nbsp;&nbsp;cloudMetadataStore CloudMetadataStore, \
&nbsp;&nbsp;fileCredentialStore jujuclient.CredentialStore, \
&nbsp;&nbsp;clientStore jujuclient.ClientStore, \
&nbsp;&nbsp;apiRoot api.Connection, \
&nbsp;&nbsp;newCloudAPIFunc func(base.APICallCloser) CloudAPI, \
&nbsp;&nbsp;newClientConfigReaderFunc func(string) (clientconfig.ClientConfigFunc, error), \
) cmd.Command { \
... \
}